src/delta_lake/delta_service.py

```diff
@@ -33,6 +35,57 @@
     "vacuum",
 }
 
+DISALLOW_SQL_META_CHARS = {
+    "--",
+    "/*",
+    "*/",
+    ";",
+    "\\",
+}
+
+ALLOWED_STATEMENTS = {
+    "select",
+}
+
+FORBIDDEN_POSTGRESQL_SCHEMAS = {
+    "pg_",
+    "pg_catalog",
+    "information_schema",
+}
```

Comment:
Are there any deltalake schemas or tables that should be forbidden?

Reply:
I'm assuming here there's 0 chance we'll ever want to run this on MySQL or some other db engine that has different tables that need to be disallowed

Reply:
not anything planned. But it could change - we will need to adjust this setting.

Reply:
Added a note, good enough for now